mitogen/parent.py

```diff
@@ -148,6 +148,9 @@ def _ioctl_cast(n):
 
 IS_LINUX = os.uname()[0] == 'Linux'
 
+IS_SOLARIS = os.uname()[0] == 'SunOS'
```

Comment:
How much can we rely on different Illumos distributions sticking to 'SunOS'?

Reply:
In that case, because it's the "base" (if you will) OS, and there's no real hierarchy in `uname` all the derived versions have kept it for compatibility.  It's more of a platform specifier at this point.